schema/users.yml

```diff
@@ -0,0 +1,11 @@
+users:
+  user_type:
+    - string
+    - required
+    - source type of the user
+    - example: 'GORKANA | NGC3'
+  user_id:
+    - string|integer
+    - required
+    - id of the user
```

Comment:
From which system is this user_id? This file seems to be missing role assignments as well. Are those no longer required?

Reply:
I'll have to follow up with @EdwardHBS on this @sieglerz

Reply:
how about other fields firstname ,lastname etc they are not required any more ?

Reply:
@sieglerz @mjainrock4 Turns out that including the `users.yml` file in this schema was a mistake, so I will delete the file.